pageserver/src/tenant/layer_map.rs

```diff
@@ -249,17 +249,6 @@ impl<L> LayerMap<L>
 where
     L: ?Sized + Layer,
 {
-    ///
-    /// Find the latest layer that covers the given 'key', with lsn <
-    /// 'end_lsn'.
-    ///
-    /// Returns the layer, if any, and an 'lsn_floor' value that
```

Comment:
I do not think that removing documentation comment for public function is good idea.
If i is deteriorated, then it should be updated but not removed.

Reply:
I put it back, and added a short note that it doesn't look into 'open' and 'frozen' layers instead.